res/controllers/Vestax-VCI-100MKII-scripts.js

```diff
@@ -118,14 +127,44 @@ VCI102.jog = function(ch, midino, value, status, group) {
     }
 };
 
-VCI102.rate = [0, 0, 0, 0];
+VCI102.rate7bit = [64, 64, 64, 64];
+
```

Comment:
This variable is for raw MIDI values of 7bit-MSB. For the default will never be used, anything do. But the choice of 64 (= 0x40, center) is natural for a tentative value of pitch fader, I think.